src/frontend/components/PublicPausedLiveVideo/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Clock, Layer, Paragraph } from 'grommet';
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import videojs from 'video.js';
+
+import { H2 } from '../Headings';
+import { Video } from '../../types/tracks';
+import { useAsyncEffect } from 'utils/useAsyncEffect';
+import { resumeLive } from '../../utils/resumeLive';
+
+const messages = defineMessages({
+  pausedSince: {
+    defaultMessage: 'The webinar is in paused since ',
+    description: 'Information about how many time the webinar is paused.',
+    id: 'components.PublicPausedLive.pausedSince',
+  },
+  title: {
+    defaultMessage: 'Webinar is paused',
+    description: 'Title for the public paused live component',
+    id: 'components.PublicPausedLive.title',
+  },
+  text: {
+    defaultMessage:
+      'The webinar is paused. When resumed, the video will start again.',
+    description: 'Message for the public paused live component',
+    id: 'components.PublicPausedLive.message',
+  },
+});
+
+interface PublicPausedLiveVideoProps {
+  video: Video;
+  videoNodeRef: HTMLVideoElement;
+}
+
+export const PublicPausedLiveVideo = ({
+  video,
+  videoNodeRef,
+}: PublicPausedLiveVideoProps) => {
+  const [time, setTime] = useState<string>();
+
+  useEffect(() => {
+    if (video.live_info?.paused_at) {
+      const now = DateTime.now();
+      const pausedAt = DateTime.fromSeconds(Number(video.live_info?.paused_at));
+      const diff = now
+        .diff(pausedAt, ['hours', 'minutes', 'seconds'])
```

Comment:
I think you can directly use 
```suggestion
        pausedAt.diffNow( ['hours', 'minutes', 'seconds'])
```

Reply:
The idea is good and I would like to use your suggestion but unfortunately this is not what I want.
With your suggestion, I have a negative diff `{ hours: -0, minutes: -0, seconds: -5}` leading to an unwanted behaviour of `Clock`. The computed duration made 2 lines after is not valid anymore.